website/views/main/filters.jade

```diff
@@ -21,7 +21,7 @@
         li.filters-edit(ng-show='_editing')
           form.hrpg-input-group
             input(type='text', ng-model='_newTag', placeholder=env.t('newTag'))
```

Comment:
And I had lost a line from the filtersCtrl section too, the $scope._newTag = {name:''}; line. :-1: